### PR TITLE
Feat/implement price fetch proxy v2

### PR DESCRIPTION
**Pull request by mchappell**
# Checklist

- [x] JIRA -[LW-9533](https://input-output.atlassian.net/browse/LW-9553)
- [x] Proper tests implemented
- [x] Screenshots added.

---

## Proposed solution

Update token pricing, and fix tooltip currency issue #528 

## Testing

All tests should still pass, values between `main` and this branch should remain the same, or check them against the new api for completeness

## Screenshots

|  Before  |  After  | 
|---|---|
| image |  image |
| image |  image | 
| image |  image |
| image |  image | 

[LW-9533]: https://input-output.atlassian.net/browse/LW-9533?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by rhyslbw — COMMENTED** [review]
Looks good @mchappell, left some comments about the reminder for updating the URL once the reverse proxy is updated. cc @gytis-ivaskevicius

File: apps/browser-extension-wallet/manifest.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
...
   "host_permissions": ["<all_urls>"],
   "content_security_policy": {
-    "extension_pages": "default-src 'self' $LOCALHOST_DEFAULT_SRC; frame-src https://connect.trezor.io/ https://www.youtube-nocookie.com; script-src 'self' 'wasm-unsafe-eval' $LOCALHOST_SCRIPT_SRC; font-src 'self' https://use.typekit.net; object-src 'self'; connect-src $CARDANO_SERVICES_URLS $ADA_HANDLE_URLS https://coingecko.live-mainnet.eks.lw.iog.io https://muesliswap.live-mainnet.eks.lw.iog.io $LOCALHOST_CONNECT_SRC https://matomo.cw.iog.io/ https://eu.posthog.com/ https://use.typekit.net data:; style-src * 'unsafe-inline'; img-src * data:;"
+    "extension_pages": "default-src 'self' $LOCALHOST_DEFAULT_SRC; frame-src https://connect.trezor.io/ https://www.youtube-nocookie.com; script-src 'self' 'wasm-unsafe-eval' $LOCALHOST_SCRIPT_SRC; font-src 'self' https://use.typekit.net; object-src 'self'; connect-src $CARDANO_SERVICES_URLS $ADA_HANDLE_URLS https://coingecko.live-mainnet.eks.lw.iog.io https://api.muesliswap.com $LOCALHOST_CONNECT_SRC https://matomo.cw.iog.io/ https://eu.posthog.com/ https://use.typekit.net data:; style-src * 'unsafe-inline'; img-src * data:;"
```
[the remark this answers is not in the document]
Comment (rhyslbw) [reply apps/browser-extension-wallet/manifest.json:22]: Proxy has been updated

File: apps/browser-extension-wallet/src/lib/scripts/background/services/utilityServices.ts [hunk 2]
```diff
@@ -92,27 +89,17 @@ const handleChangeTheme = (data: ChangeThemeData) => requestMessage$.next({ type
 const { ADA_PRICE_CHECK_INTERVAL, SAVED_PRICE_DURATION, TOKEN_PRICE_CHECK_INTERVAL } = config();
 const fetchTokenPrices = () => {
   // `base-policy-id=&base-tokenname=` for ADA as base token
-  fetch('https://muesliswap.live-mainnet.eks.lw.iog.io/list?base-policy-id=&base-tokenname=')
+  fetch('https://api.muesliswap.com/lace/prices')
```
[the remark this answers is not in the document]
Comment (rhyslbw) [reply apps/browser-extension-wallet/src/lib/scripts/background/services/utilityServices.ts:92]: Proxy is updated

[the remark this answers is not in the document]
Comment (mchappell) [reply apps/browser-extension-wallet/src/lib/scripts/background/services/utilityServices.ts]: updated [9a32899](https://github.com/input-output-hk/lace/pull/838/commits/9a328996ce7fbb121fe6278a08d22536a5882452)

File: apps/browser-extension-wallet/manifest.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (mchappell) [reply apps/browser-extension-wallet/manifest.json]: updated [9a32899](https://github.com/input-output-hk/lace/pull/838/commits/9a328996ce7fbb121fe6278a08d22536a5882452)

**Review by rhyslbw — APPROVED** [review]

**Pull request merged**